server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleIndex.java

```diff
@@ -491,8 +524,32 @@ private static void setPagination(SearchOptions options, SearchRequestBuilder es
     return terms;
   }
 
+  public Set<String> listTags(String organizationUuid, @Nullable String query, int size) {
```

Comment:
Currently they are not ordered: https://next.sonarqube.com/sonarqube/api/rules/tags . I'll notify lambda about this unexpected divergence to api/issues/tags .